charts/kcp/templates/etcd-ca.yaml

```diff
@@ -1,40 +1,62 @@
-{{- if .Values.etcd.enabled -}}
+{{- if .Values.certificates.etcd.pki -}}
```

Comment:
I'm thinking about this ... does it make sense to not check `.Values.etcd.enabled` anymore? Are there use cases where you disable the etcd included in his Helm chart but you still want certificates to be generated? Otherwise I'd say all those conditions (for all the other cert things too) should check `and .Values.etcd.enabled .Values.certificates.etcd.<Relevant Key>`.

Reply:
I'm thinking about kine. You still need certificates but no etcd.